src/PatchOrchestrationApplication/CoordinatorService/src/HealthManagerHelper.cs

```diff
@@ -41,5 +41,34 @@ class HealthManagerHelper
             ServiceHealthReport serviceHealthReport = new ServiceHealthReport(serviceUri, healthInformation);
             fabricClient.HealthManager.ReportHealth(serviceHealthReport);
         }
+
+        /// <summary>
+        /// Posts a health report against the current service to indicate status of Patch Orchestration Agent
+        /// </summary>
+        /// <param name="fabricClient">Fabric client object to carry out HM operations</param>
+        /// <param name="serviceUri">Uri of the service against which health report is to be posted</param>
+        /// <param name="sourceId">SourceId of the health report</param>
+        /// <param name="healthReportProperty">Property of the health report</param>
+        /// <param name="description">Description of the health report</param>
+        /// <param name="healthState">HealthState for the health report</param>
+        /// <param name="timeToLiveInMinutes">Time to live in minutes for health report</param>
+        internal static void UpdateHealthReport(FabricClient fabricClient, Uri serviceUri, string sourceId, string healthReportProperty, string description,
```

Comment:
>UpdateHealthReport [](start = 29, length = 18)

Whats the difference between this and above method?

Reply:
It updates the sourceID as well